src/text/index.tsx

```diff
@@ -58,8 +61,10 @@ const Text = ({
     content = replaceNewLineWithBR(children)
   }
 
+  const attrRole = !isEmpty(role) ? { role } : {}
+
   return (
-    <Tag className={cc([baseClassName, displayClassName, className])}>
+    <Tag {...attrRole} className={cc([baseClassName, displayClassName, className])}>
```

Comment:
you can put role with null as a default value and put it directly on Tag?

Reply:
If we want to prevent misuses of this component, we might as well have 'alert' as the default value

Reply:
It's defined as alert for the snackbar, but it shouldn't be the default for the Text component ;)